### PR TITLE
Add tailwind

### PR DESCRIPTION
**Pull request by celestehorgan**
Adds tailwind to this project as the CSS framework of choice. For now I'm keeping around some of the old bulma-configured SASS for reference however.

### CODE REVIEW

**Comment by celestehorgan** [comment]
I know I can merge this on my own but can you take a look anyways @lucperkins? I want to make sure I didn't make any fundamental mistakes I might regret later on.

File: config.toml [hunk 1]
```diff
@@ -3,6 +3,7 @@ disableKinds = ["taxonomy", "taxonomyTerm"]
 copyright = "Cloud Native Computing Foundation"
 
 enableGitInfo = true
+theme = "oci-tailwind"
```
Comment (lucperkins) [inline config.toml]: We *could* do this as a theme, but personally I find that creating a theme makes the directory structure trickier to navigate without providing much benefit.

If you'd like the theme to be reusable across projects, however, that's a different story!

File: layouts/_default/baseof.html [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
-    <title>
-      {{ block "title" . }}
-        {{ site.Title }}
-      {{ end }}
-    </title>
-    {{ partial "css.html" . }}
-    {{ partial "favicon.html" . }}
-  </head>
+<html lang='{{ .Site.LanguageCode }}'>
+    {{- partial "head.html" . -}}
```
Comment (lucperkins) [inline layouts/_default/baseof.html]: Can we do two spaces and enforce that in the `.editorconfig` file?

File: layouts/partials/head.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<meta charset="utf-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1" />
+<title>{{ block "title" . }}
+{{- .Title }} - {{ .Site.Title -}}
```
Comment (lucperkins) [inline layouts/partials/head.html]: `.Site` can be replaced with `site` (which enables you to avoid trickery like `$.Page.Site.foo`

**Comment by lucperkins** [comment]
Just some little nits that you can freely ignore

**Comment by caniszczyk** [comment]
@lucperkins @celestehorgan can you confirm this is only tailwind css (which is MIT licensed) and not tailwind ui which is non open source? I just want to be wary we don't leak anything commercial into a fully open source project

**Comment by lucperkins** [comment]
Yes, we’ve discussed this behind the scenes

File: layouts/_default/baseof.html (hunk 2 above)
Comment (celestehorgan) [reply layouts/_default/baseof.html]: I'll switch to VS code for you, Luc ;)

File: config.toml (hunk 1 above)
Comment (celestehorgan) [reply config.toml]: I'll have a think on it. I like the idea of doing overrides to a default in a seperate location but as I continue I'm finding weird idiosyncronisities with the approach – namely that configuring `tailwind.config.js` only seems to work in the theme dir and not the main hugo dir's.

**Pull request merged**